hail/python/hail/methods/statgen.py

```diff
@@ -652,31 +662,35 @@ def logistic_regression_rows(test, y, x, covariates, pass_through=()) -> hail.Ta
     mt = matrix_table_source('logistic_regresion_rows/x', x)
     check_entry_indexed('logistic_regresion_rows/x', x)
 
-    analyze('logistic_regresion_rows/y', y, mt._col_indices)
+    y_is_list = isinstance(y, list)
+    if y_is_list and len(y) == 0:
+        raise ValueError(f"'logistic_regression_rows': found no values for 'y'")
 
-    all_exprs = [y]
     for e in covariates:
-        all_exprs.append(e)
-        analyze('logistic_regression/covariates', e, mt._col_indices)
+        analyze('logistic_regression_rows/covariates', e, mt._col_indices)
 
-    _warn_if_no_intercept('logistic_regresion_rows', covariates)
+    _warn_if_no_intercept('logistic_regression_rows', covariates)
 
     x_field_name = Env.get_uid()
-    y_field_name = '__y'
+    y_field = list(f'__y_{i}' for i in range(len(y))) if y_is_list else "__y"
+
+    y_dict = dict(zip(y_field, y)) if y_is_list else {y_field: y}
+    func = Env.hail().methods.LogisticRegression
```

Comment:
My goal here was to preserve much of the original code path for single phenotype logistic regression.  My rational is that this function is probably widely used and this approach seems less risky.  The alternative looks like it will require more refactoring, which probably should be done after some burn in time.

Reply:
I just mean that `func` is only used once below (is that right?) so it could be inlined there. There aren't two different function names being called like in linear regression.

This is fine though.